examples/asset/processing/asset_processing.rs

```diff
@@ -240,6 +241,7 @@ fn setup(mut commands: Commands, assets: Res<AssetServer>) {
         c: assets.load("foo/c.cool.ron"),
         d: assets.load("d.cool.ron"),
         e: assets.load("embedded://asset_processing/e.txt"),
+        f: assets.load("foo/f.cool.ron"),
```

Comment:
This needs a comment explaining that defaults can be set up at a folder level. Ideally *all* of these variants should have a comment explaining why they're in the example, but that's fine.